README.md

```diff
@@ -6,16 +6,28 @@
 
 Support for Spine-based Java apps running under Google App Engine.
 
-To start a localhost emulator and run tests run `./gradlew check`.
-To start an emulator without running tests `./gradlew startDatastore`.
-To stop datastore use standard system means (e.g. `kill -9 \$(lsof -i:8080)`).
-To run the task successfully, you must have `gcloud` tool properly
-installed and configured: 
+
+##### Testing
+
+###### Preconditions
+
+To run the task successfully, you must have `gcloud` tool properly installed and configured: 
  - install gcloud of the last version
```

Comment:
Please add either `;` or `,` at the end of the items.
The last item in the list should end with a dot.